uprotocol/communication/simplenotifier.py

```diff
@@ -45,16 +46,23 @@ def __init__(self, transport: UTransport):
             raise ValueError(UTransport.TRANSPORT_NOT_INSTANCE_ERROR)
         self.transport = transport
 
-    async def notify(self, topic: UUri, destination: UUri, payload: Optional[UPayload] = None) -> UStatus:
+    async def notify(
+        self, topic: UUri, destination: UUri, options: Optional[CallOptions] = None, payload: Optional[UPayload] = None
+    ) -> UStatus:
         """
         Send a notification to a given topic.
 
         :param topic: The topic to send the notification to.
         :param destination: The destination to send the notification to.
+        :param options: Call options for the notification.
         :param payload: The payload to send with the notification.
         :return: Returns the UStatus with the status of the notification.
         """
         builder = UMessageBuilder.notification(topic, destination)
+        if options:
```

Comment:
Only comment. if options isn't none, are we guaranteed to have priority, timeout and token? if not, does trying to get those values when they don't exist cause an issue? thinking if options exists but priority, timeout or token aren't set

Reply:
CallOptions has default values for all the fields

Reply:
perfect